.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
Why 2?

Reply:
No particular reason, just seems to be a popular choice. Some of the `.c` files seem to be using 2 spaces ([example](https://github.com/bytecodealliance/wasmtime/blob/master/examples/gcd.c)) and some of the `.wat` files also use 2 spaces ([example](https://github.com/bytecodealliance/wasmtime/blob/master/cranelift/wasmtests/icall.wat)). However, sometimes there is an inconsistent mix of 2 and 4 spaces ([example](https://github.com/bytecodealliance/wasmtime/blob/66aec23a0bb45a0bbe53eb7e9f722fd691af483e/cranelift/wasmtests/simd.wat#L8-L18)).

I think it would be fine to leave `indent_size` unset if the choice is controversial.

Reply:
I think either way is fine. Our plan is to rewrite the remaining handful of .cpp files into Rust anyway :-).